pixeltable/exprs/expr.py

```diff
@@ -483,3 +496,95 @@ def _make_arithmetic_expr(self, op: ArithmeticOperator, other: object) -> 'pixel
             return ArithmeticExpr(op, self, Literal(other))  # type: ignore[arg-type]
         raise TypeError(f'Other must be Expr or literal: {type(other)}')
 
+    def _make_applicator_function(self, fn: Callable, col_type: Optional[ts.ColumnType]) -> 'pixeltable.func.Function':
+        """
+        Creates a unary pixeltable ``Function`` that encapsulates a python ``Callable``. The result type of
```

Comment:
Just FYI, I think single backticks are sufficient.

Reply:
Ok. I think I was parroting something else I saw in the codebase. Anyway, I switched to single backticks. Thanks!